test/functional/sqlsrv/MsHelper.inc

```diff
@@ -410,6 +539,201 @@ function fetchAll($conn, $tbname)
+                     new ColumnMeta('varbinary(512)', 'c21_varbinary'),
+                     new ColumnMeta('varbinary(max)', 'c22_varbinary_max'),
+                     new ColumnMeta('image', 'c23_image'),
+                     new ColumnMeta('uniqueidentifier', 'c24_uniqueidentifier'),
+                     new ColumnMeta('datetime', 'c25_datetime'),
+                     new ColumnMeta('smalldatetime', 'c26_smalldatetime'),
+                     new ColumnMeta('timestamp', 'c27_timestamp'),
+                     new ColumnMeta('xml', 'c28_xml'),
+                     );
+
+    return createTable($conn, $tbname, $columns);
+}
+
+/**
+ * Insert the specified number of rows with test data into the test table
+ * @param  resource  $conn : connection resource
+ * @param  string  $tbname : name of the table to create
+ * @param  int  $rowCount : number of test rows to be inserted
+ * @return number of rows inserted
+ */
+function insertTestRows($conn, $tbame, $rowCount)
+{
+    $count = 0;
+    for ($i = 0; $i < $rowCount; $i++) {
+        if (insertTestRow($conn, $tbame, rand(1, 20))) {
+            $count++;
+        }
+    }
+
+    if ($count != $rowCount) {
+        die("$count rows inserted instead of $rowCount\n");
+    }
+    return ($count);
+}
+
+/**
+ * Insert a number of rows with test data into the test table given the indexes
+ * @param  resource  $conn : connection resource
+ * @param  string  $tbname : name of the table to create
+ * @param  int  $minIndex : index of the first row to be inserted
+ * @param  int  $maxIndex : index of the last row to be inserted
+ * @return number of rows inserted
+ */
+function insertTestRowsByRange($conn, $tbame, $minIndex, $maxIndex)
+{
+    $count = 0;
+    $rowCount = $maxIndex - $minIndex + 1;
+    if ($rowCount > 0) {
+        for ($i = $minIndex; $i <= $maxIndex; $i++) {
+            if (insertTestRow($conn, $tbame, $i)) {
+                $count++;
+            }
+        }
+    }
+    if ($count != $rowCount) {
+        die("$count rows inserted instead of $rowCount\n");
+    }
+    return ($count);
+}
+
+/**
+ * Insert one or more specific rows of test data into the test table
+ * @param  resource  $conn : connection resource
+ * @param  string  $tbname : name of the table to create
+ * @param  int  $index : the index of a certain row of test data
+ * @return object sqlsrv statement upon success or false otherwise
+ */
+function insertTestRow($conn, $tbname, $index)
+{
+    if ($index < 1 || $index > 20) {
+        echo("Invalid row index $index for test data!\n");
+        return false;
+    }
+
+    // get array of input values
+    $inputArray = getInsertArray($index);
+    if (empty($inputArray)) {
+        fatalError("getInsertSqlComplete: inputs for inserting a row cannot be empty");
+    }
+    
+    $result = null;
+    if (isColEncrypted()) {
+        $stmt = insertRow($conn, $tbname, $inputArray, $result, INSERT_PREPARE_PARAMS);
```

Comment:
INSERT_PREPARE_PARAMS is redundant since in insertRow, isColEncrypted() and INSERT_PREPARE_PARAMS go to the same path (line 435)